doc/manual/src/release-notes/rl-next.md

```diff
@@ -1,2 +1,6 @@
 # Release X.Y (202?-??-??)
 
+- Speed-up of downloads from binary caches. The number of parallel downloads
+  (also known as substitutions) has been separated from the [`--max-jobs` setting](../command-ref/conf-file.html#conf-max-jobs). It
+  is now set to 16 by default (before this change the default was set to 1). The
```

Comment:
This is confusing because it's ambiguous what "it" refers to. I read it as if max jobs is now 16 and had to check.
What you probably mean is the number of substitution jobs. That sentence should be after mentioning the new name.

It would be easier for me to review and make suggestions if every sentence was on one line instead of breaking lines arbitrarily, because then I wouldn't have to write a screen page full of prose and just show what I mean instead.